tasks/populate-release-notes-images/populate-release-notes-images.yaml

```diff
@@ -44,11 +44,11 @@ spec:
         for ((i = 0; i < $NUM_COMPONENTS; i++))
         do
             component=$(jq -c --argjson i "$i" '.components[$i]' "${SNAPSHOT_FILE}")
-            name=$(jq -r '.name' <<< $component)
-            repo=$(jq -r '.repository' <<< $component)
-            deliveryRepo=$(translate-delivery-repo $repo | jq -r '.[] | select(.repo=="redhat.io") | .url')
-            tags=$(jq -c '.tags' <<< $component)
-            image=$(jq -r '.containerImage' <<< $component)
+            name=$(jq -r '.name' <<< "$component")
+            rh_registry_repo=$(jq -r '."rh-registry-repo"' <<< "$component")
+            deliveryRepo=$rh_registry_repo
```

Comment:
Why this + the line before it instead of just
`deliveryRepo=$(jq -r '."rh-registry-repo"' <<< "$component")`?

Reply:
ah yes, thanks!